.vib/zookeeper/goss/goss.yaml

```diff
@@ -0,0 +1,22 @@
+file:
+  /opt/bitnami/zookeeper/logs:
+    exists: true
+    filetype: directory
+    mode: "0775"
+  /opt/bitnami/zookeeper/conf/zoo.cfg:
+    exists: true
+    filetype: file
+    mode: "0644"
+    contains:
+      - {{ printf "/clientPort=.*%s/" .Env.ZOO_PORT_NUMBER }}
+command:
+  user-id-test:
+    exec: if [ "$(id -u)" -eq 0 ]; then exit 1; fi
+    exit-status: 0
+    stdout: []
+    stderr: []
+  zookeeper-version-ok:
```

Comment:
You are missing one more test to have a minimum of 5. I recommend you also test if it is possible to create a zNode using the CLI.